base/compiler/tfuncs.jl

```diff
@@ -1568,6 +1568,38 @@ end
 
 # Query whether the given intrinsic is nothrow
 
+function _nthbit(x, n)
```

Comment:
Should we make the `@nospecialize` explicit for these functions? (IIRC, it is already applied to the whole file)